themes/ddbasic/sass/module/_availability.scss

```diff
@@ -46,36 +46,36 @@
     margin-bottom: 5px;
     padding: 0;
 
-    &.pending {
+    &.pending { // grey
```

Comment:
I would prefer the inline comment's to be on it's own line an as an sentence. 

E.g.

<code>
// Sets the reservation color to "grey".
</code>